apps/allocations/app/components/Card/Budget.js

```diff
@@ -0,0 +1,213 @@
+            </ContextMenuItem>
+          </ContextMenu>
+        </MenuContainer>
+        <CardTitle theme={theme}>{name}</CardTitle>
+        <StatsContainer>
+          <StyledStats>
+            <StatsValueBig theme={theme}>
+              <Text>Inactive</Text>
+            </StatsValueBig>
+          </StyledStats>
+        </StatsContainer>
+      </StyledCard>
+    )
+  }
+
+  return (
+    <StyledCard screenSize={screenSize}>
+      <MenuContainer>
+        <ContextMenu>
+          <ContextMenuItem onClick={newAllocation}>
+            <IconPlus />
+            <ActionLabel>New Allocation</ActionLabel>
+          </ContextMenuItem>
+          <ContextMenuItem onClick={edit}>
+            <IconEdit />
+            <ActionLabel>Edit</ActionLabel>
+          </ContextMenuItem>
+          <ContextMenuItem onClick={deactivate}>
+            <IconProhibited />
+            <ActionLabel>Deactivate</ActionLabel>
+          </ContextMenuItem>
+        </ContextMenu>
+      </MenuContainer>
+      <CardTitle theme={theme}>{name}</CardTitle>
+      <StatsContainer>
+        <StyledStats>
+          <StatsValueBig theme={theme}>
+            {displayCurrency(BigNumber(amount))}
+            <Text>{' ' + currency + ' per period'}</Text>
+          </StatsValueBig>
+          <StatsValueBig css={{ paddingTop: '24px' }} theme={theme}>
+            <ProgressBar
+              color={String(theme.accentEnd)}
+              value={BigNumber(allocated).div(amount).toNumber()}
+            />
+          </StatsValueBig>
+          <StatsValueSmall css={{
+            color: theme.content,
+            paddingTop: '8px',
+          }}>
+            {displayCurrency(BigNumber(amount).minus(allocated))}
+            <Text>{' ' + currency + ' below limit'}</Text>
+          </StatsValueSmall>
+          <StatsValueSmall css={{
+            color: theme.contentSecondary,
+            paddingTop: '4px',
+          }}>
+            {BigNumber(amount)
+              .minus(allocated)
+              .div(amount)
+              .multipliedBy(100)
+              .dp(0)
+              .toString()}
+            <Text>{'% remaining'}</Text>
+          </StatsValueSmall>
+        </StyledStats>
+      </StatsContainer>
+    </StyledCard>
+  )
+}
+
+Budget.propTypes = {
+  id: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  amount: PropTypes.string.isRequired,
+  currency: PropTypes.string.isRequired,
+  allocated: PropTypes.string.isRequired,
+  inactive: PropTypes.bool.isRequired,
+  onNewAllocation: PropTypes.func.isRequired,
+  onEdit: PropTypes.func.isRequired,
+  onDeactivate: PropTypes.func.isRequired,
+  onReactivate: PropTypes.func.isRequired,
+  screenSize: PropTypes.number.isRequired,
+}
+
+const StyledCard = styled(Card)`
+  display: flex;
+  margin-bottom: 2rem;
+  margin-right: ${props => props.screenSize < CARD_STRETCH_BREAKPOINT ? '0.6rem' : '2rem'};
+  box-shadow: 0px 2px 4px rgba(221, 228, 233, 0.5);
```

Comment:
Do we have an aragon DS color definition for this shadow color?

Reply:
Yes we do. Thank you for pointing that out.